airbyte-cdk/bulk/core/base/src/testFixtures/kotlin/io/airbyte/cdk/command/CliRunner.kt

```diff
@@ -64,8 +64,9 @@ data object CliRunner {
                 AirbyteDestinationRunner(
                     args,
                     testProperties,
+                    isTest = true,
```

Comment:
Isn't `isTest = true` implied by the presence of `testBeanDefinitions`? If so get rid of it

Reply:
oh derp, good point. will do that.

Reply:
Feel free to dismiss my review after doing this, I don't want to block you

Reply:
done